.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   root: true,
-  extends: ['eslint-config-digitalbazaar'],
+  extends: ['eslint-config-digitalbazaar/jsdoc'],
```

Comment:
should be using `eslint-config-digitalbazaar` at 2, which means ['digitalbazaar', 'digitalbazaar/jsdoc'] here.